app/Platform/database/seeders/RoleSeeder.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Platform\Database\Seeders;
+
+use Illuminate\Database\Seeder;
+use Spatie\Permission\Models\Role;
+use Spatie\Permission\Models\Permission;
+
+class RoleSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     */
+    public function run(): void
+    {
+       $admin = Role::create(['name' => 'Admin']);
+       $member = Role::create(['name' => 'Member']);
```

Comment:
It would be useful if we use [PHP Enums](https://www.php.net/manual/en/language.types.enumerations.php) for stuff like roles so that we can reused.